frontend/src/components/_common/Subway/SubwayLineIcon/SubwayLineIcon.tsx

```diff
@@ -1,44 +1,48 @@
 import styled from '@emotion/styled';
 
-import { flexCenter, title4 } from '@/styles/common';
+import { flexCenter } from '@/styles/common';
 import SUBWAY_LINE_PALLETE, { SubwayLineName } from '@/styles/subway';
 
+type Size = 'small' | 'medium';
 interface Props {
   lineName: SubwayLineName;
+  size?: Size;
 }
 
-const SubwayLineIcon = ({ lineName }: Props) => {
+const SubwayLineIcon = ({ lineName, size = 'medium' }: Props) => {
   const lineColor = SUBWAY_LINE_PALLETE[lineName];
 
   const isNumberTypeSubwayName = lineName.slice(-2) === '호선' && lineName.length === 3;
 
   return (
-    <S.Box color={lineColor} isCircle={isNumberTypeSubwayName}>
-      <S.Text>{isNumberTypeSubwayName ? lineName.slice(0, lineName.length - 2) : lineName}</S.Text>
+    <S.Box size={size} color={lineColor} isCircle={isNumberTypeSubwayName}>
+      <S.Text size={size}>{isNumberTypeSubwayName ? lineName.slice(0, lineName.length - 2) : lineName}</S.Text>
     </S.Box>
   );
 };
 
 const S = {
-  Box: styled.span<{ color: string; isCircle: boolean }>`
+  Box: styled.span<{ color: string; isCircle: boolean; size: Size }>`
     display: inline-block;
 
-    width: ${({ isCircle }) => isCircle && '2rem'};
-    height: 2rem;
+    width: ${({ isCircle, size }) =>
+      isCircle && size === 'medium' ? '2rem' : isCircle && size === 'small' ? '1.4rem' : null};
```

Comment:
```typescript
const sizeMap = {small:'1.4rem', medium:'2rem'};
```
해놓고
```suggestion
    width: ${({ isCircle, size }) => isCircle && sizeMap[size]};
```
로 간결화할수있을거같네요

Reply:
훨씬 좋네요!!👍👍